src/utils/WithToggleControl.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+/**
+ * Provides boolean isChecked prop to wrapped component.
+ * Prop name can also be customized if need be
+ *
+ * @param {React.element} ToggleComponent - the component to wrap
+ * @returns {React.element}
+ */
+export default function withToggleControl(WrappedComponent, propName = 'isChecked') {
```

Comment:
See my comment here about simplifying this with a consistent prop name: https://github.com/meetup/meetup-web-components/pull/217/files?diff=unified#r118526489